index.js

```diff
@@ -4,13 +4,14 @@ const {stringToBytes, readUInt64LE, tarHeaderChecksumMatches, uint8ArrayUtf8Byte
 const xpiZipFilename = stringToBytes('META-INF/mozilla.rsa');
 const oxmlContentTypes = stringToBytes('[Content_Types].xml');
 const oxmlRels = stringToBytes('_rels/.rels');
+const zipHeader = [0x50, 0x4B, 0x3, 0x4];
```

Comment:
Can you revert this change? Doesn't really make sense here as it's not used anywhere else.